packages/orm/src/index.ts

```diff
@@ -117,7 +118,7 @@ interface Flavoring<FlavorT> {
 }
 export type Flavor<T, FlavorT> = T & Flavoring<FlavorT>;
 
-export function setField(entity: Entity, fieldName: string, newValue: any): void {
+export function setField(entity: Entity, fieldName: string, newValue: any): boolean {
```

Comment:
Not a big fan of just returning an opaque boolean from a function that isn't implicitly obvious what it is from the function declaration.  

Not sure it's a huge deal, though, since this is basically private anyways.